test/vstest/runtests.cmd

```diff
@@ -0,0 +1,7 @@
+set target=%1
```

Comment:
Starting to think we should have `runtests.cmd` and `runtests.ps1`, and make the cmd a dummy script that starts powershell.